doc/application_plugins/rlm_perl.rst

```diff
@@ -11,6 +19,46 @@ the following packages to be installed:
 * libtry-tiny-perl
 * libjson-perl
 
+The module itself may be downloaded at [#rlmPerl]_ and placed at, e.g.,
+``/usr/share/privacyidea/freeradius/privacyidea_radius.pm``.
+
+Setup
+-----
+
+Then you need to configure your FreeRADIUS site and the perl module. The
+latest FreeRADIUS plugin uses the ``/validate/check`` REST API of privacyIDEA.
+
+You need to configure the perl module in FreeRADIUS ``modules/perl`` to look
+something like this::
+
+   perl {
+       module = /usr/share/privacyidea/freeradius/privacyidea_radius.pm
+   }
+
+Your freeradius enabled site config should contain something like this::
+
+   authenticate {
+        Auth-Type Perl {
+           perl
+        }
+        digest
+        unix
+   }
+
+While you define the default authenticate type to be ``Perl`` in the
+``users`` file::
+
+   DEFAULT Auth-Type := Perl
+
+.. note:: The privacyIDEA module uses other perl modules that were not thread
+   safe in the
+   past. So in case you are using old perl dependencies and are experiencing
+   thread problems, please start FreeRADIUS with the -t switch.
+   (Everything works fine with Ubuntu 14.04 and Debian 7.)
```

Comment:
How does this fit here?

Did you experience any issues or need to use the -t switch?
Maybe we should rework this paragraph, since it is in contrast to the statement above?